voila/handler.py

```diff
@@ -21,6 +21,11 @@
 from .execute import executenb, VoilaExecutePreprocessor
 from .exporter import VoilaExporter
 
+try:
+    TimeoutError  # Py 3
+except NameError:
+    TimeoutError = RuntimeError  # Py 2
```

Comment:
Do we still need this?

Reply:
Probably not, I will do some cleaning of python2 code, thanks.

Reply:
:+1: